source/features/repo-age-first-commit.gql

```diff
@@ -0,0 +1,16 @@
+query GetFirstCommit($owner: String!, $name: String!) {
```

Comment:
I'm trying to keep the gql files next to their features so that they're clearly linked. This decision isn't final. 

The issue is that we're importing .gql files as raw text at the moment so we can't have multiple queries in one yet.

This file is the result of both points.

Reply:
Having one file per feature sounds like the most maintainable solution for me. (And it could be checked automatically in the feature validator, which could prevent mistakes like the one mentioned in my previous comment.)

But having multiple queries and importing them as `import {Query1, Query2} from './feature.gql'` would of course be helpful.